docker/dockerfiles/be-ubuntu.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# This docker file build the Starrocks be ubuntu image
+# Please run this command from the git repo root directory to build:
+# DOCKER_BUILDKIT=1 docker build --rm=true -f docker/dockerfiles/be-ubuntu.Dockerfile -t starrocks/be-ubuntu:tag .
+
+ARG ARTIFACTIMAGE=artifact:latest
+FROM ${ARTIFACTIMAGE} as artifacts
+
+FROM ubuntu:22.04
+
+RUN apt-get update -y \
+        && apt-get install -y --no-install-recommends binutils-dev default-jdk python2 \
+           mysql-client curl vim tree net-tools \
+        && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME=/lib/jvm/default-java
+
+ARG STARROCKS_ROOT=/opt/starrocks
+
+WORKDIR $STARROCKS_ROOT
+
+# Copy all artifacts to the runtime container image
```

Comment:
Whether if possible to add a new stage `artifacts-local` stage, it takes all the above operation as base image, and copy from docker context directly (namely `output/fe`, `output/be`) directly instead of artifacts, so it will be convenient for developers to build and test their binaries quickly enough in one stop command (with `--target xxx` parameter to `docker build`), rather than two commands (first to build artifacts `artifacts.Dockerfile` and then build the deployment images `all-ine-one.Dockerfile`)

Reply:
I have added support for packing runtime container with locally built artifacts.